packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.component.js

```diff
@@ -9,11 +9,40 @@ import Inject from '../../Inject';
 import getDefaultT from '../../translate';
 import TooltipTrigger from '../../TooltipTrigger';
 
-function TitleSubHeader({ title, iconId, loading, inProgress, editable, getComponent, ...rest }) {
+function TitleSubHeader({
+	title,
+	iconId,
+	loading,
+	inProgress,
+	editable,
+	getComponent,
+	onEdit,
+	onCancel,
+	onSubmit,
+	...rest
+}) {
+	const [edit, setEdit] = React.useState(false);
```

Comment:
Maybe `isEditMode` or `isEdit` could be more fitting as a boolean/state variable name? (just because my first impression of `{!edit ? <SubTitle {...rest} /> : null}` was "what exactly does edit mean here?")